book/version_control.md

```diff
@@ -395,140 +336,102 @@ To handle a data breach, you should:
 
-The basic elements of an issue are the:
-* Title and description, provided by the person that submitted the issue
-* Labels to categorise issues (e.g. bug)
-* Comments, where others can discuss the issue
-* Assignees that are working on resolving the issue
+Along side the main "Conversation" view of a PR, the GitHub web interface also provides a useful overview of changes that are included in the PR. You're able to view the list of individual commits that are included in the PR and, better still, you can view changes to each file in detail.
 
-Issues can be linked to specific Pull Requests (below) that resolve or help to solve the issue.
-They can also reference other related issues (e.g. `#12`), both within the repo and between repos.
+```{figure} ./_static/github_pr_changes.png
+---
+width: 90%
+name: github_pr_changes
+alt: The GitHub Pull Request web interface.
+---
+Changes from [an example Pull Request on the `fingertipsR` package](https://github.com/ropensci/fingertipsR/pull/91/files).
+```
 
-Issues are useful for discussing bugs and new features within the team, but can also be added by users.
-This is often the case with open source projects, providing users with a platform to highlight what would be most useful for them.
+In the "Files changed" section of a PR (shown above) altered sections of files are shown before and after the changes were made, on the left and right respectively. Where changes have deleted lines of code, these lines are highlighted in red on the left panel. And changes that add lines of code to the file are shown on the right.
 
-[Setting issue templates](https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/configuring-issue-templates-for-your-repository) for your project can be an effective way of encouraging collaborators to use informative descriptions.
-For example, a bug issue should include simple instructions to help maintainers reproduce the problem.
-While feature requests might include information on how the user expects the new feature to work or details what problem it will help them to overcome.
+This highlighted summary of changes provides a useful interface for [peer review](peer_review.md). When carrying out a review using this view, comments can added to specific lines of code and these comments can include suggested changes. All comments made using this view are also shown in the main Conversation view. When completing a review, a reviewer can indicate whether the PR should be merged or additional changes are required.
 
-Issues are a useful soundboard for requesting changes, but the implementation of changes are handled by Pull Requests (below).
+Once a PR has been reviewed and the reviewer is happy with the changes, the Conversation view can be used by internal developers to merge the PR changes onto the target branch.
```

Comment:
Again worth raising explicit settings you can put on protected branches so people can't merge to master without a review.